components/FeatureHighlight/tests/unit/FeatureHighlightTitleBodyAccessibilityMutatorTests.m

```diff
@@ -0,0 +1,70 @@
+//
+//  FeatureHighlightTitleBodyAccessibilityMutatorTests.m
+//  MaterialComponentsUnitTests
+//
+//  Created by Mohammad Cazi on 3/5/18.
+//
+
+#import <XCTest/XCTest.h>
+#import "MaterialFeatureHighlight.h"
+#import "MDCFeatureHighlightAccessibilityMutator.h"
+
+static NSArray<UIColor *> *testColors(){
+  return @[[UIColor whiteColor], [UIColor blackColor], [UIColor redColor], [UIColor orangeColor],
+           [UIColor greenColor], [UIColor blueColor], [UIColor grayColor]];
+}
+
+@interface FeatureHighlightTitleBodyAccessibilityMutatorTests : XCTestCase
+
+@end
+
+@implementation FeatureHighlightTitleBodyAccessibilityMutatorTests
+
+- (void)testMutateChangesTextColor {
+  for (UIColor *color in testColors()) {
+    // Given
+    MDCFeatureHighlightViewController *highlightVC =
+        [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:[UIView new]
```

Comment:
Nit: Please use `alloc] init]` instead of `new]`

Reply:
done.